tests/integration/docker-compose.yml

```diff
@@ -32,9 +32,10 @@ services:
 
   temporalite:
     container_name: temporalite
-    build:
-      dockerfile: temporalite.Dockerfile
-      context: .
+#    build:
```

Comment:
Maybe remove comments if the temporalite image is working ?

Reply:
Ok